test/features/user_can_read_rest_of_robot_statistics_test.rb

```diff
@@ -0,0 +1,193 @@
+require_relative "../test_helper"
+class UserCanReadRobotStatsDptTest < FeatureTest
+  def test_user_see_department_numbers
```

Comment:
this is a really, really long test! i'd suggest breaking it up into smaller tests if you want to test each individual department number. you could have a test to check for each individual department 👍